src/content/language/operators.md

```diff
@@ -500,6 +501,25 @@ Strictly speaking, the "double dot" notation for cascades isn't an operator.
 It's just part of the Dart syntax.
 :::
 
+## Spread operators
+
+The spread operator evaluates an expression that yields a collection,
```

Comment:
nit: Switching from plural to singular felt a bit jarring to me. Consider sticking with one or the other.

Reply:
Do you mean like, the heading is plural (`## Spread operators`) but the sentence is singular (`The spread operator evaluates...`)?

Reply:
Yes. As a reader unfamiliar with the spread operators it makes me wonder, what happened to the other spread operator(s)?